utils_nlp/models/transformers/common.py

```diff
@@ -43,9 +65,7 @@ def fine_tune(
             if fp16:
                 with amp.scale_loss(loss, optimizer) as scaled_loss:
                     scaled_loss.backward()
-                torch.nn.utils.clip_grad_norm_(
-                    amp.master_params(optimizer), args.max_grad_norm
-                )
+                torch.nn.utils.clip_grad_norm_(amp.master_params(optimizer), args.max_grad_norm)
             else:
                 loss.backward()
                 torch.nn.utils.clip_grad_norm_(model.parameters(), args.max_grad_norm)
```

Comment:
I think we need to switch the order of lines 75 and 76, otherwise, you'll get a warning. See https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate